### PR TITLE
Eswe 453 wrs mvp work profile back end change status endpoint new

### PR DESCRIPTION
**Pull request by sacintha**
Please do not merge . I have a few more tests to do . Mostly sanity, But I need a review from you

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/TestData.kt [hunk 1]
```diff
@@ -1,17 +1,19 @@
 package uk.gov.justice.digital.hmpps.educationemploymentapi.service
 
 import com.vladmihalcea.hibernate.type.json.internal.JacksonUtil
-import org.mockito.kotlin.mock
+// import org.mockito.kotlin.mock
```
Comment (theDustRoom) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/TestData.kt:4]: Can you remove the commented imports

File: src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/TestData.kt [hunk 2]
```diff
@@ -20,17 +22,15 @@ import uk.gov.justice.digital.hmpps.educationemploymentapi.data.jsonprofile.Work
 import uk.gov.justice.digital.hmpps.educationemploymentapi.data.jsonprofile.WorkInterests
 import uk.gov.justice.digital.hmpps.educationemploymentapi.data.jsonprofile.WorkTypesOfInterest
 import uk.gov.justice.digital.hmpps.educationemploymentapi.entity.ReadinessProfile
-import uk.gov.justice.digital.hmpps.educationemploymentapi.repository.ReadinessProfileRepository
+// import uk.gov.justice.digital.hmpps.educationemploymentapi.repository.ReadinessProfileRepository
```
Comment (theDustRoom) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/TestData.kt:25]: And this one

**Review by theDustRoom — CHANGES_REQUESTED** [review]
Looks good, just a little cleanup you missed

File: src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/TestData.kt (hunk 1 above)
Comment (sacintha) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/TestData.kt]: removed

File: src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/TestData.kt (hunk 2 above)
Comment (sacintha) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/TestData.kt]: removed

File: Work readiness.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,626 @@
...
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "signin",
+			"request": {
+				"auth": {
+					"type": "basic",
+					"basic": {
+						"password": "v6oP++I2dUO3.z,F2ahKcUM,8o<oGSnV(8ylu,C)SiOF7img!&kIw9a.(noH",
```
Comment (theDustRoom) [inline Work readiness.postman_collection.json:14]: You should use a variable for your credentials in Postman so they're not checked into a public repository.

**Review by theDustRoom — CHANGES_REQUESTED** [review]
Just a minor change to remove your credentials from the Postman collection

**Review by theDustRoom — APPROVED** [review]
LGTM

File: Work readiness.postman_collection.json (hunk 3 above)
Comment (sacintha) [reply Work readiness.postman_collection.json:14]: ok will change

**Review by theDustRoom — APPROVED** [review]
LGTM

**Pull request merged**